src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  padding: 0 16px;
+  justify-content: center;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
```

Comment:
```suggestion
```

redundant styles. block elements already in column by default

Reply:
After deleting   "flex-direction: column; and display: flex; " the image moved from the central position a bit, so I had to add extra { margin-right: 3px;} to the card__image to make it aligned.